README.md

```diff
@@ -38,6 +38,12 @@ The app is designed to give you an idea of what SwiftCurrent can do with minimal
 
 [We're working on it now!](https://github.com/wwt/SwiftCurrent/milestone/2)
 
+If you would like to try the beta release, please install the `BETA_SwiftCurrent_SwiftUI` product in SPM or the `BETA_SwiftUI` sub spec in CocoaPods.  For more detailed steps, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).  The "Getting Started with SwiftUI" can be [found here](https://github.com/wwt/SwiftCurrent/blob/main/wiki/Getting-Started-SwiftUI.md).  The SwiftUI example app can be found under the ExampleApp folder.  To run it, clone our repo, open `SwiftCurrent.xcworkspace`, target the `SwiftUIExampleApp` scheme, and run to a simulator.
+
+In order to use the library with SwiftUI, your minimum targeted versions must meet: iOS 14.0, macOS 11, tvOS 14.0, and watchOS 7.0.
```

Comment:
```suggestion
In order to use the library with SwiftUI, your minimum targeted versions must meet: iOS 14.0, macOS 11, tvOS 14.0, or watchOS 7.0.
```

This might be clearer or more confusing. I'm on the fence about it because I should know that if I'm only building for iOS that I only need to worry about iOS 14.0 min. At the same time this reads as though I need to meet all of them even if only building for iOS. This would also be the same as how it's stated in Installation

Reply:
+1 this suggestion

Reply:
Thank you.  I fixed it in Installation and missed it here.